assets/src/edit-story/components/panels/pageAdvancement.js

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import { useCallback } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../app/story';
+import { Switch, Numeric, Row, Label } from '../form';
+import RangeInput from '../rangeInput';
+import Note from './shared/note';
+import { SimplePanel } from './panel';
+
+const SwitchRow = styled.div`
+  margin-bottom: 16px;
+`;
+
+const BoxedNumeric = styled(Numeric)`
+  padding: 6px 6px;
+  border-radius: 4px;
+`;
+
+const DEFAULT_AUTO_ADVANCE = true;
+const DEFAULT_PAGE_DURATION = 7;
+
+function PageAdvancementPanel() {
+  const {
+    state: {
+      story: {
+        autoAdvance = DEFAULT_AUTO_ADVANCE,
+        defaultPageDuration = DEFAULT_PAGE_DURATION,
+      },
+    },
+    actions: { updateStory },
+  } = useStory();
+
+  const handleChangeValue = useCallback(
+    (prop) => (value) => updateStory({ properties: { [prop]: value } }),
```

Comment:
Could we make the actual update to Story only once the user has stopped sliding? Or perhaps have a delay for making the actual update? Thoughts?

Reply:
Some debouncing makes sense 👍